contracts/finality/src/state/public_randomness.rs

```diff
@@ -10,6 +10,17 @@ use cw_storage_plus::{Bound, Map};
 /// Map of public randomness values by fp public key and block height
 pub(crate) const PUB_RAND_VALUES: Map<(&[u8], u64), Vec<u8>> = Map::new("pub_rand_values");
```

Comment:
Should this become private?

Reply:
These const methods are usually public, but why not, since we have all of those getters / setters.